rebar.config

```diff
@@ -12,8 +12,22 @@
              {pretty_print, erl_pp},
              {preprocess, true}]}.
 
+{deps_error_on_conflict, true}.
+
 {deps, [
-    {eredis,"1.*", {git, "https://github.com/Nordix/eredis.git", {tag, "v1.4.1"}}},
-    {ecredis_crc16, ".*", {git, "git@github.com:HalloAppInc/ecredis-crc16.git", {branch, "master"}}},
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager", "3.6.10"}}
+    {eredis, "1.*", {git, "https://github.com/Nordix/eredis.git", {tag, "v1.4.1"}}},
+    {ecredis_crc16, ".*", {git, "https://github.com/HalloAppInc/ecredis-crc16.git", {branch, "master"}}},
+    {lager, ".*", {git, "https://github.com/erlang-lager/lager", {tag, "3.6.10"}}}
     ]}.
+
+%% see README here: https://github.com/blt/port_compiler
+{overrides,
```

Comment:
why do we need to add this?
rebar3 also compiles native code right? - plugin states that it adds the rebar2 support for this..
i think im missing something.

Reply:
This is for one of this project's dependencies, ecredis-crc16 – that project is using rebar2 still